index.html

```diff
@@ -1,9 +1,9 @@
 <!doctype html>
 <h1>Web Bluetooth Demos</h1>
 <ul>
-  <li><a href="https://webbluetoothcg.github.io/demos/bluetooth-toy-plane/">bluetooth-toy-plane</a></li>
```

Comment:
The full path is used to force https redirect when folks find http://webbluetoothcg.github.io/demos.